.travis.yml

```diff
@@ -1,29 +1,52 @@
 language: java
+
 jdk:
   - oraclejdk8
-sudo: false
+
+sudo: required
+
+services:
+  - docker
+
+env:
+  DOCKER_COMPOSE_VERSION: 1.11.2
+
+addons:
+  hosts:
+    - hotstorage
+
 before_cache:
   - rm -f  $HOME/.gradle/caches/modules-2/modules-2.lock
   - rm -fr $HOME/.gradle/caches/*/plugin-resolution/
-  - rm -fr $HOME/.gradle/caches/*/scripts/
+
 cache:
   directories:
     - $HOME/.gradle/caches/
     - $HOME/.gradle/wrapper/
 
 before_install:
-  - ./gradlew downloadDependencies
+  - sudo apt-get update
+  - sudo apt-get -y -o Dpkg::Options::="--force-confnew" install docker-engine
```

Comment:
Change this to `docker-ce` instead of `docker-engine`. @nivemaham Can you do it whenever you have time? I don't have access to do it. The Travis Build is failing because of this.